proxy/src/http/websocket.rs

```diff
@@ -159,6 +166,7 @@ async fn ws_handler(
     config: &'static ProxyConfig,
     cancel_map: Arc<CancelMap>,
     session_id: uuid::Uuid,
+    sni_hostname: Option<String>,
```

Comment:
Is it different from `host` 2 lines below? Why sni_hostname is better for SQL-over-HTTP queries?

Reply:
Yes, and SNI host is the only one that is checked against our certs, so we will reject connection with a forged SNI host.